python/foglamp/services/core/server.py

```diff
@@ -771,10 +774,41 @@ def start(cls):
                 try:
                     response = json.loads(result)
                     response['message']
-                    _logger.info("Successfully shut down the %s service %s.", svc._type, svc._name)
+                    _logger.info("Shutdown scheduled for %s service %s at %s", svc._type, svc._name, response['message'])
                 except KeyError:
                     raise
 
+    @classmethod
+    async def poll_microservices_unregister(cls):
+        """ poll microservice shutdown endpoint for non core micro-services"""
+        try:
+            shutdown_threshold = 0
+            found_services = ServiceRegistry.get()
+            cls._service_shutdown_threshold = 5 * (len(found_services) - 2)
```

Comment:
I think we should not keep it as `cls.` class level variable but local variable.

Reply:
ok